src/mcpbr/cli.py

```
@@ -576,6 +576,24 @@ def run(
+        import docker
+
+        docker_client = docker.from_env()
+        docker_client.ping()
+        docker_info = docker_client.info()
+        server_version = docker_info.get("ServerVersion", "unknown")
+        console.print(f"[green]✓ Docker validation passed: Docker {server_version} running[/green]")
+    except docker.errors.DockerException as e:
+        console.print(f"[red]✗ Docker validation failed: Docker not available: {e}[/red]")
+        console.print("[yellow]Make sure Docker Desktop is running[/yellow]")
+        sys.exit(1)
+    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)